### ISSUE TITLE
Recent gpodder 3.10.20 PPA update gives "youtube-dl" extension error

### ISSUE THREAD

**Issue by rizlaw** [gpodder/gpodder#1069]
![gpodder 3 10 20 error](https://user-images.githubusercontent.com/65782169/122078476-9f062500-cdca-11eb-980c-9e98b5cd9149.png)
Received PPA update to gpodder yesterday. I was hoping this would fix inability to download youtube podcasts for the last several weeks (I have youtube setup correctly with my key in gpodder). Every time I open the new version of gpodder I get the error popup: 

> Your version of youtube-dl 2020.03.24 has know issues, please upgrade to 2021.02.04 or newer.

Since gpodder is supposed to have the "youtube-dl" extension embedded in the application, I don't see any way to  "upgrade" the extension within gpodder.

Running Ubuntu 20.04.2 LTS / Gnome

**Comment by auouymous** [comment]
Youtube-dl is only bundled with the Mac and Windows builds of gpodder. You either have an old version on your system or the PPA version of gpodder bundles an old version and you'll need to contact whoever made it.

Or disable the youtube-dl extension if you don't need special formats, more than 15 episodes or DRM/age-restricted content.

**Comment by auouymous** [comment]
@elelay https://launchpad.net/~gpodder/+archive/ubuntu/ppa lists you as the uploader, is youtube-dl bundled with it?

**Comment by elelay** [comment]
yes I upload them. No I don't include youtube-dl. `pip install --user youtube_dl` is the best way to go on ubuntu in my opinion.

**Comment by elelay** [comment]
ubuntu focal only has the old version you mention: https://packages.ubuntu.com/focal/youtube-dl
